src/modules/middleware/blacklistMiddleware.ts

```diff
@@ -0,0 +1,18 @@
+import { Request, Response, NextFunction } from 'express';
+import { logger, Logger } from '../common/Logger';
+
+/**
+ * Middleware to check the blacklist IP
+ *
+ */
+export const isBlackList = async (req: Request, res: Response, next: NextFunction) => {
```

Comment:
The option must be available according to the config settings.
If there is no config set value or DB connection fails,
It must pass to the default value(next).

Reply:
resolved